charts/flux2/templates/kustomize-controller.yaml

```diff
@@ -36,6 +36,17 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        {{- if .Values.kustomizecontroller.envFrom.enabled }}
+        envFrom:
+          {{- if .Values.kustomizecontroller.envFrom.map.enabled }}
+          - configMapRef:
+              name: {{ .Values.kustomizecontroller.envFrom.map.name }}
+          {{- end }}
+          {{- if .Values.kustomizecontroller.envFrom.secret.enabled }}
```

Comment:
Should be replaced by:
{{- if .Values.kustomizecontroller.envFrom.secret.name }}